pages/playlists/index.tsx

```diff
@@ -17,6 +17,18 @@ const Playlist: NextPage = () => {
     isLoadingMore,
   } = usePaginatedPlaylists();
 
+  if (!playlists) {
+    return (
+      <div className="flex h-screen">
```

Comment:
You can remove the extra `div` on line 23 if you add the following

```suggestion
      <div className="flex h-screen justify-center items-center">
```